pkg/reconciler/revision/controller.go

```diff
@@ -115,7 +117,12 @@ func newControllerWithOptions(
 		logger.Info("Fetch GitHub commit ID from kodata failed", zap.Error(err))
 	}
 
-	digestResolveQueue := workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "digests")
+	digestResolveQueue := workqueue.NewNamedRateLimitingQueue(workqueue.NewMaxOfRateLimiter(
+		workqueue.NewItemExponentialFailureRateLimiter(20*time.Millisecond, 30*time.Minute),
```

Comment:
I personally fin 30m a bit high to be honest. At that point, I'd probably think the system is hung or something :joy:. An y reason to make the ceiling **that** high? I would've thought like 1m to be more appropriate than the default 1s.

Reply:
I think the default is 1000 seconds = 15ish minutes (https://github.com/kubernetes/client-go/blob/master/util/workqueue/default_rate_limiters.go#L41), but I guess 15 vs 30 doesn't really make much difference, bumping the base is much more important (and then after that maybe the QPS too 🤔). Will move back to 1000s, bump base to 1s, and fix modules.txt after lunch.

Reply:
Oh I misread that then, then I agree, the diff prolly doesn't matter.

Reply:
updated, PTAL